index.d.ts

```diff
@@ -40,7 +40,7 @@ declare namespace koaBody {
         /**
          * {String} If you want checksums calculated for incoming files, set this to either 'sha1' or 'md5', default false
          */
-        hash?: string;
```

Comment:
What's the purpose of this change? This would break existing `koa-body` implementations in production.

Reply:
The option.hash in formidable has been renamed into option.hashAlgorithm. I can make an adaptation in order to not release a major version.